### PR TITLE
qr

### LINKED ISSUE #2702: qr

### PR DESCRIPTION
**Pull request by karalleyna**
Closes #2702

### Remark
I completed #2702 because there has been no activity for 1 week and the PR has already been closed.

### CODE REVIEW

**Comment by karalleyna** [comment]
@sharique1006  Could you review my PR? I couldn't find where tests fail. Hence, I will update my PR according to your suggestions.

Screen Shot 2022-09-17 at 20 41 36

**Comment by karalleyna** [comment]
Hi @sreegandh804 ,

Could you check it out?

**Comment by sharique1006** [comment]
Hi, @vaatsalya123. This PR is for jax.lax.linalg.qr. You seemed to have committed shift_right_logical in this. Can you please move that to a different PR?

**Comment by sharique1006** [comment]
Hi @karalleyna, a lot of changes from other commits are reflected here which are not required for `qr`. I would suggest you to create a new PR for your function and I can close this one.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
